src/pages/landing/stocks/_footerBanner.js

```diff
@@ -0,0 +1,205 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Flex } from 'components/containers'
+import { Header, QueryImage } from 'components/elements'
+import { LinkButton } from 'components/form'
+import { localize } from 'components/localization'
+import device from 'themes/device.js'
+
+const BannerWrapper = styled(Flex)`
+    overflow: hidden;
+    border-top: 1px solid rgba(151, 151, 151, 0.2);
+    background-color: var(--color-black);
+    padding: 53px 120px 56px;
+    height: 100%;
+
+    @media ${device.tabletL} {
+        height: unset;
+        padding: 16px;
+    }
+`
+const MainWrapper = styled(Flex)`
+    width: 100%;
+    max-width: 1200px;
+    margin: 0 auto;
+    flex-direction: row;
+    align-items: center;
+    position: relative;
+    @media ${device.desktopL} {
+        max-width: 1600px;
+    }
+    @media ${device.tabletL} {
+        flex-direction: column;
+    }
+`
+const TextWrapper = styled(Flex)`
+    width: 60%;
+    justify-content: start;
+
+    @media ${device.tabletL} {
+        height: unset;
+        width: 100%;
+        justify-content: center;
+        align-items: center;
+        padding: 24px 0;
+    }
+`
+const TextDiv = styled(Flex)`
+    width: 75%;
+    max-width: 62rem;
+    position: relative;
+    text-align: left;
+    flex-direction: column;
+    justify-content: start;
+    align-items: center;
+    z-index: 2;
+
+    @media ${device.tabletL} {
+        max-width: 100%;
+        text-align: center;
+        justify-content: center;
+    }
+`
+
+const ImageWrapper = styled(Flex)`
+    width: 40%;
+
+    @media ${device.tabletL} {
+        height: unset;
+        width: 100%;
+        align-items: center;
+        padding: 36px 0;
+    }
+`
+const ImageDiv = styled(Flex)`
+    width: 488px;
+    align-items: center;
+
+    @media ${device.tabletL} {
+        max-width: 300px;
+    }
+    @media ${device.tablet} {
+        height: unset;
+    }
+`
+const BtnDiv = styled(Flex)`
+    width: 100%;
```

Comment:
This can be omitted as it defaults to 100%.

Reply:
omitted in the latest commit.